content/pt-br/docs/setup/production-environment/tools/kubespray.md

```diff
@@ -0,0 +1,125 @@
+
+* **A versão mínima exigida do Kubernetes é v1.22**
+* **Ansible v2.11+, Jinja 2.11+ and python-netaddr estão instalados na máquina que executará os comandos do Ansible**
+* Os servidores de destino devem ter acesso à Internet para extrair imagens do docker. Caso contrário, é necessária configuração adicional. Consulte ([Offline Environment](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/offline-environment.md))
+* Os servidores de destino são configurados para permitir o **encaminhamento IPv4**.
+* Se estiver usando IPv6 para pods e serviços, os servidores de destino serão configurados para permitir o **encaminhamento de IPv6**.
+* Os **firewalls não são gerenciados**, você precisará implementar suas próprias regras da maneira que costumava fazer. Para evitar qualquer problema durante o deployment, você deve desabilitar seu firewall.
+* Se o kubespray for executado a partir de uma conta de usuário não raiz, o método correto de escalonamento de privilégios deve ser configurado nos servidores de destino. Em seguida, o `ansible_become` flag ou os parâmetros de comando `--become` ou `-b` devem ser especificados.
+
+Kubespray fornece as seguintes utilidades para ajudar a provisionar seu ambiente:
+
+* Scripts do [Terraform](https://www.terraform.io/) para os seguintes provedores de nuvem:
+  * [AWS](https://github.com/kubernetes-sigs/kubespray/tree/master/contrib/terraform/aws)
+  * [OpenStack](https://github.com/kubernetes-sigs/kubespray/tree/master/contrib/terraform/openstack)
+  * [Equinix Metal](https://github.com/kubernetes-sigs/kubespray/tree/master/contrib/terraform/metal)
+
+### (2/5) Componha um arquivo de inventário
+
+Depois de provisionar seus servidores, crie um [arquivo de inventário para o Ansible](https://docs.ansible.com/ansible/latest/network/getting_started/first_inventory.html). Você pode fazer isso manualmente ou por meio de um script de inventário dinâmico. Para mais informações, consulte "[Criando seu próprio inventário](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/getting-started.md#building-your-own-inventory)".
+
+### (3/5) Planeje o deployment do seu cluster
+
+O Kubespray oferece a habilidade de customizar vários aspectos do deployment:
+
+* Escolha do modo de deployment: kubeadm ou non-kubeadm
+* Plugins CNI (rede)
+* Configuração de DNS
+* Escolha da camada de gerenciamento: nativo/binário ou conteinerizado
+* Versões de componentes
+* Reflexões de rota Calico
+* Opções de agente de execução do componente
+  * {{< glossary_tooltip term_id="docker" >}}
+  * {{< glossary_tooltip term_id="containerd" >}}
+  * {{< glossary_tooltip term_id="cri-o" >}}
+* Métodos de geração de certificados
+
+As customizações do Kubespray podem ser feitas em um [arquivo variável](https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html). Se você estiver começando com o Kubespray, considere usar os padrões do Kubespray para implantar seu cluster e explorar o Kubernetes.
```

Comment:
A variable file, in the context of Ansible, Terraform, and other infrastructure as code tools is a file that holds values for variables. So "arquivo de variáveis" would be more precise in this context.

Reply:
Thanks, I didn't know about that.